docs/user_guides/debugging/index.rst

```diff
@@ -45,18 +45,18 @@ Running GDB and creating breakpoint in kernel:
 .. code-block:: bash
 
     $ gdb-oneapi -q --args python simple_sum.py
-    (gdb) break simple_sum.py:22  ### Set breakpoint in kernel
+    (gdb) break simple_sum.py:8  ### Set breakpoint in kernel
     (gdb) run
     ...
-    Thread 2.2 hit Breakpoint 1, with SIMD lanes [0-7], __main__::data_parallel_sum () at simple_sum.py:22
+    Thread 2.2 hit Breakpoint 1, with SIMD lanes [0-7], __main__::data_parallel_sum () at simple_sum.py:8
     8           i = dppy.get_global_id(0)
     (gdb) next
-    23           c[i] = a[i] + b[i]
+    9           c[i] = a[i] + b[i]
     (gdb) continue
     Done...
     ...
 
-If breakpoint does not work and you see in output
+If breakpoint is not hit and you see the following output:
```

Comment:
s/If breakpoint is not hit and you see the following output/If the breakpoint is not hit, you will see the following output/

Reply:
Fixed